explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -179,7 +180,16 @@ export function UpgradeableProgramDataSection({
         {account.details?.space !== undefined && (
           <tr>
             <td>Data (Bytes)</td>
-            <td className="text-lg-end">{account.details.space}</td>
+            <td className="text-lg-end">
+              <Downloadable
+                data={programData.data[0]}
```

Comment:
I'd prefer if this downloaded binary data instead of a base64 string

Reply:
Conversion done in the `Downloadable` component. Two observations:

1. `Downloadable` is probably early, halfway optimization but I did not want to:
* pollute `UpgradeableLoaderAccountSection`, nor
* needlessy make `Downloadable` as generic as it should be (since it's a new otherwise unused component anyway)
2. the downloaded data differs by a few leading bytes from what gets downloaded through `solana account`. I'm not 100% sure what part of the data that is, but I'm guessing the generic attributes that already get parsed? (`lamports`/`executable`/etc)

Let me know if you have issue with these points or if this will do.

Reply:
The difference is due to the [45 bytes of metadata](https://github.com/solana-labs/solana/blob/3bd5a89d6ff680bdfc2a64b30081ecd46548d19a/sdk/program/src/bpf_loader_upgradeable.rs#L286) which is stored before the executable in the account. When using `solana account`, the full raw data of an account is downloaded.

```
> solana account 99hNSg9acmyg5hWTjw76GoA8k3LG5NvdPXih7sotdmZ7 -o 99hNSg9acmyg5hWTjw76GoA8k3LG5NvdPXih7sotdmZ7.bin --url testnet
> ls -l
total 5808
-rw-r--r--  1 starry  staff  1485581 Jan 16 11:49 99hNSg9acmyg5hWTjw76GoA8k3LG5NvdPXih7sotdmZ7.bin
-rw-r--r--@ 1 starry  staff  1485536 Jan 16 11:50 bytecode-99hNSg9acmyg5hWTjw76GoA8k3LG5NvdPXih7sotdmZ7.txt
```

The `data` field of `programData` is populated by RPC using this account parser: https://github.com/solana-labs/solana/blob/3bd5a89d6ff680bdfc2a64b30081ecd46548d19a/account-decoder/src/parse_bpf_loader.rs#L10 and you can see that it takes into account the "offset" of where the executable data actually starts.

Long story short, your change is correct! And also, doing the conversion in `Downloadable` looks good to me.